common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/ClientCertAuthChallengeHandler.java

```diff
@@ -626,4 +631,31 @@ public void stopYubiKitManagerUsbDiscovery() {
         //Should be called when host fragment is destroyed.
         mYubiKitManager.stopUsbDiscovery();
     }
+
+    /**
+     * Telemet results from certificate based authentication (CBA) if CBA occurred.
+     * @param response a RawAuthorizationResult object received upon a challenge response received.
+     */
+    public void telemetCertBasedAuthResults(@NonNull final RawAuthorizationResult response) {
+        if (mIsOnDeviceCertBasedAuthProceeding || mIsSmartcardCertBasedAuthProceeding) {
+            //Telemet Results based on which type of CBA occurred.
+            final CertBasedAuthEvent certBasedAuthEvent;
+            if (mIsOnDeviceCertBasedAuthProceeding) {
+               certBasedAuthEvent =  new CertBasedAuthEvent(TelemetryEventStrings.Event.CERT_BASED_AUTH_ON_DEVICE_EVENT);
```

Comment:
nit: maybe rename to CERT_BASED_AUTH_ON_DEVICE_RESULT_EVENT and CERT_BASED_AUTH_SMARTCARD_RESULT_EVENT